### PR TITLE
(#854) Update to contributing.md 

### PR DESCRIPTION
**Pull request by KrisBierma**
Dead and inactive links updated to help new devs sign the agreement and 
communicate about issues. Git commands added and some sentences 
reworded for clarity.

Issue #854

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,37 +1,39 @@
...
   * You must do this for each Chocolatey project that requires it.
   * If you are curious why we would require a CLA, we agree with Julien Ponge - take a look at his [post](https://julien.ponge.org/blog/in-defense-of-contributor-license-agreements/).
 
 ## Contributing Process
 
 ### Get Buyoff Or Find Open Community Issues/Features
 
- * Through GitHub, or through the [mailing list](https://groups.google.com/forum/#!forum/chocolateygui) (preferred), you talk about a feature you would like to see (or a bug), and why it should be in Chocolatey.
-   * If approved through the mailing list, ensure the accompanying GitHub issue is created with information and a link back to the mailing list discussion.
+ * Through [GitHub discussions](https://github.com/chocolatey/ChocolateyGUI/discussions) you talk about a feature you would like to see (or a bug), and why it should be in Chocolatey.
```
Comment (gep13) [inline CONTRIBUTING.md:19]:
```suggestion
 * Through [GitHub discussions](https://github.com/chocolatey/ChocolateyGUI/discussions) you talk about a feature you would like to see (or a bug), and why it should be in Chocolatey GUI.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
+   * If approved, ensure the accompanying [GitHub issue](https://github.com/chocolatey/ChocolateyGUI/issues) is created with information and a link back to the GitHub discussion.
  * Once you get a nod from one of the [Chocolatey Team](https://github.com/chocolatey?tab=members), you can start on the feature.
  * Alternatively, if a feature is on the issues list with the [community tag](https://github.com/chocolatey/chocolateygui/labels/Community), it is open for a patch. You should comment that you are signing up for it on the issue so someone else doesn't also sign up for the work.
 
 ### Set Up Your Environment
 
  * You create, or update, a fork of chocolatey/chocolateygui under your GitHub account.
- * From there you create a branch named specific to the feature.
+ * Clone it to your computer. `git clone https://github.com/chocolatey/ChocolateyGUI.git`
```
Comment (gep13) [inline CONTRIBUTING.md:27]: This should be a step to clone the fork of the repository, not the original repository, right?

**Review by gep13 — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@KrisBierma thanks again for taking the time to submit this PR, really appreciate it!  I have left a couple of comments for some changes.

Can I also ask that you tag the issue in the commit message?  i.e. rather than having:

```
(doc) Update to contributing.md
```

use:

```
(#854) Update to contributing.md
```

Also, as mentioned in the doc, can you wrap the main body of the commit message at 72 characters?

Thanks again!

File: CONTRIBUTING.md (hunk 2 above)
Comment (KrisBierma) [reply CONTRIBUTING.md]: Of course. Updated to `https://github.com/your-github/ChocolateyGUI.git`. Will this work?

File: CONTRIBUTING.md (hunk 1 above)
Comment (KrisBierma) [reply CONTRIBUTING.md]: Done.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@KrisBierma thank you very much for getting this updated! Really appreciate it!

**Pull request merged**